fixture_magic/utils.py

```diff
@@ -87,3 +91,10 @@ def add_to_serialize_list(objs):
         if key not in seen:
             serialize_me.append(obj)
             seen[key] = 1
+
+
+def get_instance_model_full_name(instance):
+    return "{}.{}".format(
+        instance._meta.app_label,
+        instance._meta.model_name
+    )
```

Comment:
you can get rid of this method simply by using [`instance._meta.label`](https://github.com/django/django/blob/b68b8cb89abb35ff2152175ea540619ec384b1f4/django/db/models/options.py#L131)